build.boot

```diff
@@ -31,13 +31,13 @@
 (bootlaces! +version+)
 
 (task-options!
-  javac   {:options (let [jdk7-bootclasspath (System/getenv "JDK7_BOOTCLASSPATH")]
+  javac   {:options (let [jdk8-bootclasspath (System/getenv "JDK7_BOOTCLASSPATH")]
                       (concat
                         []
-                        (when-not (empty? jdk7-bootclasspath)
-                          ["-source" "1.7"
-                           "-target" "1.7"
-                           "-bootclasspath" jdk7-bootclasspath])))}
+                        (when-not (empty? jdk8-bootclasspath)
+                          ["-source" "1.8"
+                           "-target" "1.8"
+                           "-bootclasspath" jdk8-bootclasspath])))}
```

Comment:
I don't know boot too  well, so I was kind of shooting in the dark here :P.  I'll switch this over in a bit.

Definetly let me know if I messed anything up in here. Also, it's fine to upgrade this boot file and leave the other projects as they are for now, right?